### DISCUSSION TITLE
Coloring UMAP for specific marker not working

### DISCUSSION THREAD

**Discussion by rmauron** [GreenleafLab/ArchR#1836] [Questions / Documentation]
Whenever I run the code which is supposed to color the UMAPs regarding a specific marker/gene, the color stays monochrome. 
Any ideas of what is wrong?

**Comment by rcorces** [comment]
Please provide a reproducible example using the tutorial dataset.

**Reply by rmauron** [reply]
Hi, here are the type of figures I get using the data used in the manual:
[Plot-UMAP-Marker-Genes-RNA-W-Imputation.pdf](https://github.com/GreenleafLab/ArchR/files/10667702/Plot-UMAP-Marker-Genes-RNA-W-Imputation.pdf)
[Plot-UMAP-Marker-Genes-W-Imputation.pdf](https://github.com/GreenleafLab/ArchR/files/10667703/Plot-UMAP-Marker-Genes-W-Imputation.pdf)
[Plot-UMAP-Marker-Genes-WO-Imputation.pdf](https://github.com/GreenleafLab/ArchR/files/10667704/Plot-UMAP-Marker-Genes-WO-Imputation.pdf)

**Reply by rcorces** [reply]
Sorry - thats not a reproducible example. But might be related to this:
https://github.com/GreenleafLab/ArchR/issues/1731

**Reply by rmauron** [reply]
Sorry, I did not know I was supposed to send the .log file and version of my packages. 
However, after reading discussion #1731, it appears that with ggplot2 version 3.3.6, it works and gives me the same plot when embedding.
Thanks a lot for the fast reply and the amazing work!
